README.md

```diff
@@ -14,3 +14,10 @@ password:example_pass
 ```
 - Fetch the latest code with `git pull`
 - Update PyPI with `python setup.py sdist upload`
+
+
+## Updating histogram_tools.py
+moztelemetry/histogram_tools.py is a mirror of its counterpart from
+[mozilla-central](https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/histogram_tools.py).
+To update it to the latest version you can run `update_histogram_tools`.
+Such command is available only if you install the moztelemetry package using `python setup.py install`.
```

Comment:
As an alternative, I can wrap the update_histogram_tools in a python standalone script and put it in a bin directory at the root of the repository. Do you have a preference?